CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
+* [Windows containerd v1.7](https://github.com/Azure/AKS/issues/3975) will be the default container runtime for k8s v1.28+ on AKS Windows nodes. Windows Host Process (HPC) containers is GA in Windows containerd v1.7 and it has some [breaking changes](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support#container-mounts).
+
+### Release notes 
+
+* Bug Fixes 
+  * Fix for when an update with the identity property is issued on an existing AKS Fleet with MSI.
+  * Fix for if the system nodepool is set to be dedicated, Vertical Pod Autoscaler (VPA) pods can still be scheduled to system nodes.
+  * Fix for if VPA webhook's CABundle is different from the cert stored in secret, the VPA webhook image version will be bumped.
+  * Fix where pod age is now calculated from cluster creation time rather than cluster stop time for [managed cluster start/stop](https://learn.microsoft.com/en-us/azure/aks/start-stop-cluster?tabs=azure-cli).
+  * In AKS with Azure CNI, there is possible deadlock scenario where Container Network Service (CNS) API is not available. [Aync Delete](https://github.com/Azure/azure-container-networking/tree/master/docs/feature/async-delete) fixes this deadlock issue. The CNI calls to CNS to release an IP address from a Pod asynchronously with a failsafe in such a way that if CNS is unavailable, it can recover these events when it does eventually start.
+  * Fix for Windows NPM crashes in k8s 1.28 with Containerd 1.7. Bug was a result Windows NPM DaemonSet referencing a file that did not exist in its current directory.[Containerd 1.7](https://github.com/Azure/AKS/issues/3975).
+  * Fixed fleet clusters, so they will not be correctly set to NRG-Lockdown RestrictionLevel Restricted, instead of Unspecified. Additionally, fleet clusters within one of the undesired Unspecified states will be fixed on reconcile.
+* Behavioral Change
+  * All AKS managed namespaces now have a "kubernetes.azure.com/managedby:" AKS label. 
+  * VPA updater and recommender resource requests and limits can now be configured by customer to avoid OOMkill or resource wastage.
+  * Fix to prevent Open Service Mesh control plane from fighting with AKS admission enforcer.
```

Comment:
Needs language fix.

Corrected conflict between Open Service Mesh and AKS Admission Enforcer